<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DokumenteForm</class>
 <widget class="QWidget" name="DokumenteForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Dokumente</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButton_new">
                <property name="text">
                 <string>&amp;Neu</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_edit">
                <property name="text">
                 <string>&amp;Bearbeiten</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_delete">
                <property name="text">
                 <string>&amp;LÃ¶schen</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_period"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="FilterableTableView" name="tableView_documents"/>
          </item>
          <item row="2" column="0">
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="ClickableLabel" name="label_documentImage">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterableTableView</class>
   <extends>QTableView</extends>
   <header>lib.FilterableTableView</header>
  </customwidget>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>lib.ClickableLabel</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
